io.openems.edge.battery.bmw/src/io/openems/edge/battery/bmw/BmwBatteryImpl.java

```diff
@@ -85,19 +84,17 @@ protected void setModbus(BridgeModbus modbus) {
 	public BmwBatteryImpl() {
 		super(//
 				OpenemsComponent.ChannelId.values(), //
-				StartStoppable.ChannelId.values(), //
 				Battery.ChannelId.values(), //
+				StartStoppable.ChannelId.values(), //
 				BMWChannelId.values() //
 		);
 	}
 
 	@Activate
-	void activate(ComponentContext context, Config config) throws OpenemsException {
+	void activate(ComponentContext context, Config config) {
 		this.config = config;
-		if (super.activate(context, config.id(), config.alias(), config.enabled(), config.modbusUnitId(), this.cm,
-				"Modbus", config.modbus_id())) {
-			return;
-		}
+		super.activate(context, config.id(), config.alias(), config.enabled(), config.modbusUnitId(), this.cm, "Modbus",
+				config.modbus_id());
```

Comment:
@JohannesHundschell / @pooran-c: why was this updated/approved? It is recommended to keep it the way it was.